packages/react-components/tests/omni-panel.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { OmniPanel, OmniPanelView } from '../lib';
+import { OmniPanelProps } from '../lib/omni-panel';
+import { mockOnClick } from './test-utils';
+
+function TestPanel(props: Omit<OmniPanelProps, 'view' | 'children'>) {
+  return (
+    <OmniPanel view={0} {...props}>
+      <OmniPanelView viewId={0}></OmniPanelView>
+    </OmniPanel>
+  );
+}
+
+describe('OmniPanel component', () => {
+  let handler: ReturnType<typeof mockOnClick>;
+
+  beforeEach(() => {
+    // TextEncoder is not available in node
```

Comment:
Since we are running the tests in browser, this comment shouldn't be valid anymore.